scripts/api/data/metadatablocks/codemeta.tsv

```diff
@@ -5,30 +5,30 @@
-	softwareRequirementsName	Name	Name or title of the required software/library		text		#VALUE		FALSE	FALSE	FALSE	FALSE	FALSE	softwareRequirements	codeMeta20	
-	softwareRequirementsVersion	Version	Version of the required software/library		text		#VALUE		FALSE	FALSE	FALSE	FALSE	FALSE	softwareRequirements	codeMeta20	
-	softwareRequirementsUrl	URL	Link to required software/library	https://...	url		<a href="#VALUE" target="_blank" rel="noopener">#VALUE</a>		FALSE	FALSE	FALSE	FALSE	FALSE	softwareRequirements	codeMeta20	
-	softwareSuggestions	Software Suggestions	Optional dependencies, e.g. for optional features, code development, etc.		none			FALSE	FALSE	TRUE	FALSE	FALSE	FALSE		codeMeta20	
-	softwareSuggestionsName	Name	Name or title of the optional software/library		text		#VALUE		FALSE	FALSE	FALSE	FALSE	FALSE	softwareSuggestions	codeMeta20	
-	softwareSuggestionsVersion	Version	Version of the optional software/library		text		#VALUE		FALSE	FALSE	FALSE	FALSE	FALSE	softwareSuggestions	codeMeta20	
-	softwareSuggestionsUrl	URL	Link to optional software/library	https://...	url		<a href="#VALUE" target="_blank" rel="noopener">#VALUE</a>		FALSE	FALSE	FALSE	FALSE	FALSE	softwareSuggestions	codeMeta20	
-	permissions	Permissions	Permission(s) required to run the code (for example, a mobile app may require full internet access or may run only on wifi).		text		#VALUE	FALSE	FALSE	TRUE	FALSE	FALSE	FALSE		codeMeta20	https://schema.org/permissions
-	memoryRequirements	Memory Requirements	Minimum memory requirements.		text		#VALUE	FALSE	FALSE	TRUE	FALSE	FALSE	FALSE		codeMeta20	https://schema.org/memoryRequirements
-	processorRequirements	Processor Requirements	Processor architecture required to run the application (e.g. IA64).		text		#VALUE	FALSE	FALSE	TRUE	FALSE	FALSE	FALSE		codeMeta20	https://schema.org/processorRequirements
-	storageRequirements	Storage Requirements	Storage requirements (e.g. free space required).		text		#VALUE	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE		codeMeta20	https://schema.org/storageRequirements
-	releaseNotes	Release Notes	Link to release notes	https://...	url		<a href="#VALUE" target="_blank" rel="noopener">#VALUE</a>	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE		codeMeta20	https://schema.org/releaseNotes
-	contIntegration	Continous integration	Link to continuous integration service	https://...	url		<a href="#VALUE" target="_blank" rel="noopener">#VALUE</a>	FALSE	FALSE	TRUE	FALSE	FALSE	FALSE		codeMeta20	
-	issueTracker	Issue Tracker	Link to software bug reporting or issue tracking system	https://...	url		<a href="#VALUE" target="_blank" rel="noopener">#VALUE</a>	FALSE	FALSE	FALSE	TRUE	FALSE	FALSE		codeMeta20	
+	operatingSystem	Operating Systems	Operating systems supported (e.g. Windows 10, OSX 11.3, Android 11).		text	4		FALSE	FALSE	TRUE	FALSE	TRUE	FALSE		codeMeta20	https://schema.org/operatingSystem

```

Comment:
**question:** As far as I understood the spec, the[ compound parent field should be type "none"](https://guides.dataverse.org/en/latest/admin/metadatacustomization.html#fieldtype-definitions). Would "text" cause a problem? Would that be another valid type? Is that type important after all?

Reply:
@poikilotherm whoops! You're right. Seems like it should be "none". Sorry, I was in a hurry and just trying to get the thing to load. These TSVs are a pain in the neck! 😄